tests/UsernameTest.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
```

Comment:
potential blocker: in the interest of communicating best practices it would probably be good to use `declare(strict_types=1);` here